softdev.bib

```diff
@@ -2899,3 +2899,70 @@ @inproceedings{shahriyar14fast
     month = Oct,
     pages = "121-139"
 }
+
+@article{martin08common,
+  author = {Martin, Robert A. and Barnum, Sean},
+  title = {Common Weakness Enumeration (CWE) Status Update},
+  year = {2008},
+  issue_date = {April 2008},
+  publisher = {Association for Computing Machinery},
+  address = {New York, NY, USA},
+  volume = {XXVIII},
+  number = {1},
+  issn = {1094-3641},
+  url = {https://doi.org/10.1145/1387830.1387835},
+  doi = {10.1145/1387830.1387835},
+  journal = {Ada Lett.},
+  month = Apr,
+  pages = {88–91},
+  numpages = {4}
+}
+
+@inproceedings{evans06scalable,
+  title={A scalable concurrent malloc (3) implementation for FreeBSD},
+  author={Evans, Jason},
+  booktitle={BSDCan},
+  year={2006},
+  month=May,
+  url={https://people.freebsd.org/~jasone/jemalloc/bsdcan2006/jemalloc.pdf}
+}
+
+@inproceedings{lietar19snmalloc,
+  author = {Li\'{e}tar, Paul and Butler, Theodore and Clebsch, Sylvan and Drossopoulou, Sophia and Franco, Juliana and Parkinson, Matthew J. and Shamis, Alex and Wintersteiger, Christoph M. and Chisnall, David},
+  title = {Snmalloc: A Message Passing Allocator},
+  year = {2019},
+  month = Jun,
+  isbn = {9781450367226},
+  publisher = {Association for Computing Machinery},
+  address = {New York, NY, USA},
+  url = {https://doi.org/10.1145/3315573.3329980},
+  doi = {10.1145/3315573.3329980},
+  booktitle = {ISMM},
+  pages = {122–135},
+  numpages = {14},
+  keywords = {Memory allocation, message passing},
+  location = {Phoenix, AZ, USA},
+  series = {ISMM 2019}
+}
+
+@misc{lea96memory,
+  title={A memory allocator},
+  author={Lea, Doug and Gloger, Wolfram},
+  year={1996},
+  url={http://web.mit.edu/sage/export/singular-3-0-4-3.old/omalloc/Misc/dlmalloc/malloc.ps}
+}
+
+@misc{gloger06wolfram,
```

Comment:
Do we need to cite this as well as the Lea/Wolfram paper?

Reply:
This is for `xalloc`, the one above (`lea96memory`) is for `dlmalloc`.

Reply:
That web page only seems to mention ptmalloc?

Reply:
Whoops, I somehow looked at the bottom comment.

I would say so, as this has links for `ptmalloc`, even though no text. I don't know what else we'd cite for this allocator.

Reply:
Since we don't use ptmalloc in the paper, I suggest we remove this reference from the PR. Later on, we can work out if we can find something for xmalloc.

Reply:
I see that we do use `ptmalloc` [1]; am I mistaken?

[1] https://github.com/capablevms/cheri_misidioms/blob/master/cheri_misidioms.ltx#L307-L308

Reply:
Oops, yes, we do use ptmalloc! OK, so let's fix this reference and use the title `ptmalloc` (otherwise it'll continue to baffle us).

Reply:
Changed in https://github.com/softdevteam/softdevbib/pull/86/commits/bb939b3a233679b4d37690992d200ade52f4ec9c.

Reply:
I agree with bb939b3, but I think we should also change the literal `title` to something like `title = "ptmalloc"`, because "Wolfram's homepage" conveys no useful information.

Reply:
Done in https://github.com/softdevteam/softdevbib/pull/86/commits/a99423fb66b1f78d90c0bc69f3481297b042ab6e (also added `\urldate` for this entry).